source/taup/intro.rst

```diff
@@ -0,0 +1,49 @@
+简介
+====
+
+:软件名称: TauP
+:主页: https://www.seis.sc.edu/taup/
+:源码地址: https://github.com/crotwell/TauP
+:作者: `Philip Crotwell <https://sc.edu/study/colleges_schools/artsandsciences/earth_ocean_and_environment/our_people/directory/crotwell_philip.php>`__
+:最新版本: v2.4.5 (2017-11-03)
+:适用平台: Linux、macOS、Windows
+:编程语言: Java
+:许可协议: GPL
+
+:教程作者: `seismo-learn <https://seismo-learn.org>`__
+:教程更新日期: 2020-12-28
```

Comment:
I think we need to update the date if we add more things later, right?

Reply:
okay~